clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -2474,6 +2480,11 @@ class AnnotatingParser {
           Current.getNextNonComment()->isOneOf(tok::comma, tok::r_brace)) {
         Current.setType(TT_StringInConcatenation);
       }
+    } else if (Style.isCpp() && Current.is(tok::kw_using)) {
+      if (Current.Next && Current.Next->Next && Current.Next->Next->Next) {
+        if (Current.Next->Next->Next->isTypeName(LangOpts))
```

Comment:
IMO this is too simplistic to be very useful. See the general syntax [here](https://en.cppreference.com/w/cpp/language/type_alias). Also, this doesn't work at all if the `using` statement is in a header file.